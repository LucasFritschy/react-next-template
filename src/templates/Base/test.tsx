import { render, screen } from '@testing-library/react'

import Base from '.'

describe('<Base />', () => {
  it('should render the children', () => {
    const { container } = render(
      <Base>
        <h1>Title</h1>
      </Base>
    )

    expect(screen.getByRole('heading', { name: /title/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
