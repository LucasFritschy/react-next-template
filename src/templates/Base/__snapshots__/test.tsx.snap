// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Base /> should render the children 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  height: 100vh;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

<div
  class="c0"
>
  <div
    class=""
  >
    <h1>
      Title
    </h1>
  </div>
</div>
`;
